miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java

```diff
@@ -175,7 +175,11 @@ protected PoolDto asDto(Pool model) {
   @ResponseBody
   public PoolDto createPool(@RequestBody PoolDto pool, UriComponentsBuilder uriBuilder, HttpServletResponse response)
       throws IOException {
-    Long id = poolService.save(Dtos.to(pool));
+    Pool poolobj = Dtos.to(pool);
+    if (poolobj.getVolume() == null && poolobj.getPoolableElementViews().stream().allMatch(view -> view.getDilutionVolumeUsed() != null)) {
+      poolobj.setVolume(poolobj.getPoolableElementViews().stream().mapToDouble(view -> view.getDilutionVolumeUsed()).sum());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)